README.md

```diff
@@ -4,89 +4,89 @@ This repository is a template to start applications from as well as the SafeInsi
+- Hosted on github in a separate repository and have github workflows triggered on every PR commit. Workflows should run the below checks at a minimum:
+    - sonarqube
+    - eslint and prettier
+    - Vitest
+    - Codecov and fail if coverage is not 100%
+    - For api apps, run open-api to generate bindings and ensure files do not change.
+- Use the Nextjs framework as a starting point as initialized by: “npx create-next-app@latest” (“@rc” for now)
+    - The default options will be selected, i.e. Typescript, ESLint, src directory and app router all enabled, but Tailwind and import alias customization disabled.
+- Have the following standard npm packages installed:
+    - vitest cdk-nextjs-standalone @clerk/nextjs aws-sdk
+    - Dev packages: tsx dotenv aws-cdk-lib prettier
+- Contain a LICENSE file with the AGPL license content
+- Eslint [shall be configured](templates/common/.eslintrc.json) to include “eslint:recommended” with the following additions. Some are semi-controversial but have served us well in other projects:
+    - semi-colons are only used [in the few situations](https://eslint.org/docs/latest/rules/semi#never) where they are required.
+    - 4 space indent (also see below editorconfig). This makes code a bit more legible, while also making indentation a bit more painful to discourage deeply nested blocks.
+- Have an [.editorconfig](https://editorconfig.org) [file](templates/common/.editorconfig) with these defaults:
+    - 4 space indent
+    - Unix line endings
+    - UTF charset
+- Tests should strive to use stuff that the user “sees” like text or ARIA roles (button/checkbox/etc) instead of attributes on elements (e.g. id or class). If id is required, add a "data-testId" attribute and [target that in the specs](https://playwright.dev/docs/locators#locate-by-test-id)
+- Github Workflows configured to:
+    - Deploy to a per-PR test environment.
+    - Staging will be automatically deployed to match the \`main\` branch upon PR merge.
+    - Production will be deployed whenever a release is tagged by an member of code-deployers group
+    - A slack message will be posted to an appropriate channel whenever one of the above deploys occur
+- Contain an AUTHORS file that is updated by each person who constributes code
 
 ### Applications that have a web user interface, the following packages will be installed:
 
--   React 19 (RC currently, release estimated this summer)
--   [mantine](ttps://github.com/safeinsights)
--   [vanilla-extract](https://vanilla-extract.style)
--   [testing-library](https://testing-library.com) react with happy-dom
-    -   [react support](https://testing-library.com/docs/react-testing-library/intro/)
--   [Playwright](https://playwright.dev)
-    -   [Playwright Lighthouse](https://github.com/abhinaba-ghosh/playwright-lighthouse)
+- React 19 (RC currently, release estimated this summer)
+- [mantine](ttps://github.com/safeinsights)
+- [vanilla-extract](https://vanilla-extract.style)
```

Comment:
do we want to remove vanilla extract in favor of panda css?

Reply:
I think so, the syntax is almost the same, just swapping out `style` for `css`.  I mainly did it for so I could test in the  management app and thought it was the only user of vanilla-styled.   I forgot about the UI in the TOA though, did you use any vanilla styles there?  here's an example of the change: https://github.com/safeinsights/management-app/pull/48/files#diff-33985376a2ef0f2869d42118d20372e5d3fddb80b02852ec528e64a321c84e5aL14

Reply:
I also just discovered that just because you pull in these changes to an app, it doesn't actually remove vanilla-extract.  In the management app, i had to manually remove it in this commit https://github.com/safeinsights/management-app/pull/48/commits/7b1a7ad313eaf19b7dec403f5b259f6c0be416af  <-  is also a good example of the needed changes to migrate